<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QWidget" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget {
    background-color: #545c64;
    border: 0px;
}

QToolButton {
    padding-left: 20px;
    color: rgb(255, 255, 255);
    background-color: rgb(84, 92, 100);
    width: 120px;
    height: 55px;
    border: 0px solid;
    font-size: 14px;
    font-family: 黑体;
}

QToolButton:hover {
    background-color: rgb(67, 74, 80);
}

QToolButton:checked {
    color: rgb(255, 208, 75);
    background-color: rgb(67, 74, 80);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnUserManage">
        <property name="text">
         <string>用户管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBookManage">
        <property name="text">
         <string>图书管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnBookBorrow">
        <property name="text">
         <string>借阅管理</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true">QWidget {
    background-color: white;
}

QPushButton {
    background-color: #fff;
    border: 1px solid #dcdfe6;
    color: #fff;
    padding: 4px 20px 2px 20px;
    border-radius: 4px;
	background-color: #409eff;
    border-color: #409eff;
}

QPushButton:hover {
    color: #fff;
    background-color: #66b1ff;
    border-color: #66b1ff;
}

QPushButton:pressed {
    color: #fff;
    background-color: #409eff;
    border-color: #409eff;
    padding-left: 0;
    padding-right: 0;
}

QLineEdit {
    background-color: #fff;
    border-radius: 3px;
    border: 1px solid #dcdfe6;
    color: #606266;
    height: 25px;
    padding: 0 15px;
    width: 100%;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>用户管理</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>搜索：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSeach"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddUser">
           <property name="text">
            <string>添加用户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteUser">
           <property name="text">
            <string>删除用户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubmitUser">
           <property name="text">
            <string>提交修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>图书管理</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>搜索：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSeach_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddBook">
           <property name="text">
            <string>添加图书</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteBook">
           <property name="text">
            <string>删除图书</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSubmitBook">
           <property name="text">
            <string>提交修改</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>借阅管理</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>搜索：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSeach_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddBook_2">
           <property name="text">
            <string>添加图书</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDeleteBook_2">
           <property name="text">
            <string>删除图书</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>90</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_3"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
